adding_species_manuscript.md

```diff
@@ -27,102 +27,74 @@ preamble: |
+        - stdpopsim is a tool recently developed to provide easy access to simulation frameworks
             -   avoid re-making common models
             -   provide standard benchmarks for methods development and testing
-
-        -   and a resource for empirical researchers, eg. power analyses or sanity checks (Adrion et al. 2020)
-
-    -   feedback from 2020/2021 workshops made it clear that the prospective community included empiricists who are especially concerned with using stdpopsim for inference for individual species that mostly not already in stdpopsim catalog
-
-2.  **ELISE** Community-based expansion of the number and variety of species and their demographic scenarios included in the stdpopsim catalog is useful for both of these major goals shared by the population genetics community, methods development and inference 
-
-    -   when first published, the stdpopsim catalog included 6 species: *Homo sapiens*, *Pongo abelii*, *Canis familiaris*, *Drosophila melanogaster*, *Arabidopsis thaliana*, and *Escherichia coli.*
-
+            -   for both methods development and empirical research
+        - but so far stdpopsim has been mainly restricted to well-characterised model organisms, limiting utility for non-model organisms or model organisms in the early stages of development
+    - many potential users want to simulate their study organism, or develop inference methods for non-model organisms
+        -   feedback from 2020/2021 stdpopsim workshops emphasized the need for simulations of a wide variety of model and non-model species
+
+2.  **ELISE** Therefore this paper is intended as a resource for both methods developers and empirical researchers to develop simulations of their own species of interest, with the potential to submit the simulation framework for inclusion in the stdpopsim catalog
+    - we discuss the elements of a population genomic simulation model that realistically characterizes a species, including:
+        - input data
+        - reasonable data quality
+        - simulation frameworks (coalescent and forward?)
+    - in addition, we discuss 
+        - limitations of available data and when a species-specific simulation may be impractical or unnecessary for the intended task
+    - lastly, we discuss how these models may be integrated into the stdpopsim catalog
+        -   we present the current method for adding species and demographic scenarios to the catalog 
+        -   and clarify the required genomic resources and QC process
+
+3.   Making a popgen simulation
+    - necessary resources:
+        - genome assembly
+            - this needs to be chromosome level, or nearly so
+            - contigs are not sufficient because:
+                - lack of linkage information limits their use for inference
+                - current limitations to the Python API/command line (eg. it complicates displaying and choosing what chromosome (contig) to use)
```

Comment:
I don't think we need to dwell too much on this - we should focus on what it would really *mean* to do simulations of a bazillion contigs (not a lot, IMO) vs giving technical reasons for why it's not done in stdpopsim. I think @petrelharp had some very good text on that point that we should try to get in here.

Reply:
Good advice (and I'd actually meant to delete those last points with specific reference to stdpopsim, whoops). @izabelcavassim has made some progress on the first parts of this section at https://github.com/popsim-consortium/adding-species-manuscript/pull/5 and I think I will ask @petrelharp to incorporate some of his ideas for the "what about species without good assemblies" section.

Reply:
maybe we could have a session: "what if one of the features above required are missing":
like:

- if a good mutation rate estimation is missing ------> borrow it from the closest species one can find
- same as above for recombination rate (?)
- if a good assembly does not exist --------> select the most contiguous/largest contigs and
- if a demographic model does not exist ----> at least Ne
- if Ne does not exist ----> the closest related species?

At the end of the day, we want all those parameters to be explicitly documented/citeable such that if at some point they are updated in the literature (e.g., a study estimated a new mutation rate for the species in the catalog) they can also be updated in the catalog. I think this is the great advantage of this open-source community, and I think we could emphasize it as well.

Reply:
awesome! Agreed!